### ISSUE TITLE
Error Window object has no attribute extract_variables (Sublime Text 2)

### ISSUE THREAD

**Issue by ReTheOff** [mtxr/SQLTools#8]
Hey, awesome project/package!  Unfortunately I can't seem to run anything.  I see in the console in ST:

Traceback (most recent call last):
  File "./sublime_plugin.py", line 339, in run_
  File "./SQLTools.py", line 154, in run
  File "./SQLToolsModels.py", line 171, in execute
AttributeError: 'Window' object has no attribute 'extract_variables'

I have my connections setup and paths to mysql.  I'm on Mac, mysql command works to server in terminal.  I thought I saw that this would work with ST2, so thought I'd give this a shot.  fyi, I installed from the tar.gz file download, as it wasn't available in package control.  Anyway, any thoughts?  I'm looking forward to working with this and (mostly) ridding myself of the standard GUI tools.

Thanks,
Greg

**Comment by mtxr** [comment]
Hey @ReTheOff .

Actually I'm working in a port to ST2 version :(
For now I'm supporting just ST3 but, probably (really not sure), I will have a port to ST2 by the end of the week.

Please, be patient :D 

PS: Just curious, why do you still using ST2? I'm looking for some beta testers (are you in?) for ST2 but I can't find almost anyone around me.

**Comment by ReTheOff** [comment]
Oh no prob. In fact, I went ahead and got ST3 finally.  To be honest, I just didn't see the need to upgrade. ST2 worked so well for me.  But packages more and more are requiring ST3, so here I am. :)  I would be happy to run some tests for you though.  

So while I'm posting things.... in ST3 I have this error in console:

```
SQLTools: Query: 
Traceback (most recent call last):
  File "/Applications/Sublime Text.app/Contents/MacOS/sublime_plugin.py", line 553, in run_
    return self.run()
  File "SQLTools in /Users/greg/Library/Application Support/Sublime Text 3/Installed Packages/SQLTools.sublime-package", line 154, in run
  File "/Users/greg/Library/Application Support/Sublime Text 3/Installed Packages/SQLTools.sublime-package/SQLToolsModels.py", line 181, in execute
  File "/Users/greg/Library/Application Support/Sublime Text 3/Installed Packages/SQLTools.sublime-package/SQLToolsModels.py", line 194, in builArgs
TypeError: can only concatenate list (not "str") to list
```

**Comment by mtxr** [comment]
After a long time of evaluation about ST2 support and porting the code to python 2, I was discouraged to do it. 
Feel free to make pull requests and collaborate with the project, but I really can't implement it now. Sorry.

It will take me too much time and ST2 is being replaced day by day by ST3.

About the ST3 bug, it was fixed on issue #9. It's fully working now.

Hope you still using SQLTools and maybe help us with the port.

Thanks.
